.openhands_instructions

```diff
@@ -1,28 +0,0 @@
-OpenHands is an automated AI software engineer. It is a repo with a Python backend
```

Comment:
Since this was deleted, will the resolver find it? It doesn't seem so https://github.com/All-Hands-AI/OpenHands/blob/32ee500c6509c3916f908947dd0d250615f11486/openhands/resolver/resolve_issue.py#L406

Reply:
Maybe we should create a new issue for it

Reply:
Done -ish, here, it's expressed as a quick prompt for `oh` https://github.com/All-Hands-AI/OpenHands/issues/5830 , but its fix was to hardcode the new path... That can't be what we want for a microagent